src/um/pdh.rs

```diff
@@ -54,3 +208,338 @@ STRUCT!{struct PDH_COUNTER_PATH_ELEMENTS_W {
+    pcchBufferLength: LPDWORD, dwDetailLevel: DWORD, bRefresh: BOOL
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhEnumObjectItemsW(
+    szDataSource: LPCWSTR, szMachineName: LPCWSTR, szObjectName: LPCWSTR,
+    mszCounterList: LPWSTR, pcchCounterListLength: LPDWORD, mszInstanceList: LPWSTR,
+    pcchInstanceListLength: LPDWORD, dwDetailLevel: DWORD, dwFlags: DWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhEnumObjectItemsA(
+    szDataSource: LPCTSTR, szMachineName: LPCTSTR, szObjectName: LPCTSTR,
+    mszCounterList: LPTSTR, pcchCounterListLength: LPDWORD, mszInstanceList: LPTSTR,
+    pcchInstanceListLength: LPDWORD, dwDetailLevel: DWORD, dwFlags: DWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhMakeCounterPathW(
+    pCounterPathElements: PPDH_COUNTER_PATH_ELEMENTS_W, szFullPathBuffer: LPWSTR,
+    pcchBufferSize: LPDWORD, dwFlags: DWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhMakeCounterPathA(
+    pCounterPathElements: PPDH_COUNTER_PATH_ELEMENTS_A, szFullPathBuffer: LPSTR,
+    pcchBufferSize: LPDWORD, dwFlags: DWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhParseCounterPathW(
+    pCounterPathElements: *mut PDH_COUNTER_PATH_ELEMENTS_W, szFullPathBuffer: LPWSTR,
+    pcchBufferSize: LPDWORD, dwFlags: DWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhParseCounterPathA(
+    pCounterPathElements: *mut PDH_COUNTER_PATH_ELEMENTS_A, szFullPathBuffer: LPTSTR,
+    pcchBufferSize: LPDWORD, dwFlags: DWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhParseInstanceNameW(
+    szInstanceString: LPCWSTR, szInstanceName: LPWSTR, pcchInstanceNameLength: LPDWORD,
+    szParentName: LPWSTR, pcchParentNameLength: LPDWORD, lpIndex: LPDWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhParseInstanceNameA(
+    szInstanceString: LPCTSTR, szInstanceName: LPTSTR, pcchInstanceNameLength: LPDWORD,
+    szParentName: LPTSTR, pcchParentNameLength: LPDWORD, lpIndex: LPDWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhValidatePathW(
+    szFullCounterPath: LPCWSTR
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhValidatePathA(
+    szFullCounterPath: LPCTSTR
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhGetDefaultPerfObjectW(
+    szDataSource: LPCWSTR, szMachineName: LPCWSTR, szDefaultObjectName: LPWSTR,
+    pcchBufferSize: LPDWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhGetDefaultPerfObjectA(
+    szDataSource: LPCTSTR, szMachineName: LPCTSTR, szDefaultObjectName: LPTSTR,
+    pcchBufferSize: LPDWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhGetDefaultPerfCounterW(
+    szDataSource: LPCWSTR, szMachineName: LPCWSTR, szObjectName: LPCWSTR,
+    szDefaultCounterName: LPWSTR, pcchBufferSize: LPDWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhGetDefaultPerfCounterA(
+    szDataSource: LPCTSTR, szMachineName: LPCTSTR, szObjectName: LPCTSTR,
+    szDefaultCounterName: LPTSTR, pcchBufferSize: LPDWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhBrowseCountersW(
+    pBrowseDlgData: PPDH_BROWSE_DLG_CONFIG
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhBrowseCountersA(
```

Comment:
PdhBrowseCountersW and PdhBrowseCountersA should use PPDH_BROWSE_DLG_CONFIG_* types.